x-pack/metricbeat/module/aws/ec2/_meta/data.json

```diff
@@ -51,12 +47,12 @@
             },
             "network": {
                 "in": {
-                    "bytes": 56,
-                    "packets": 1
+                    "bytes": 615.2,
+                    "packets": 6.2
                 },
                 "out": {
-                    "bytes": 88,
-                    "packets": 1.6
+                    "bytes": 841.4,
+                    "packets": 6.8
```

Comment:
is this update intended? doesn't seem related to the PR description

Reply:
@exekias Yes I was just trying to run generate data.json to verify the change here still works 😄